src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md

```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
 
 Please refer [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) for more details.
 
+## Dropbox
+
+PDF Services APIs can connect with the Dropbox using pre-signed URLs for downloading and uploading the files to and from Dropbox location without the need for any credentials or access tokens.
+
+Please refer here for [downloading](https://www.dropbox.com/developers/documentation/http/documentation#files-get_temporary_link) and [uploading](https://www.dropbox.com/developers/documentation/http/documentation#files-get_temporary_upload_link) files from dropbox.
+
+## Azure Blob
+
+PDF Services APIs can connect with the Azure Blob using pre-signed URLs or [SAS Urls](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview) for downloading and uploading the files to and from Azure Blob without the need for any credentials or access tokens. Use [Best Practices](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview#best-practices-when-using-sas) when using SAS URLs.
```

Comment:
remove 'to and from'
without the need for any credentials or access tokens -> 'without the need for credentials or access token'

Reply:
Done.